server/app/controllers/fusor/api/v21/subscriptions_controller.rb

```diff
@@ -13,6 +13,18 @@
 module Fusor
   class Api::V21::SubscriptionsController < Api::V2::BaseController
 
+    before_filter :check_content_type, :only => :upload
+
+    def check_content_type
+      # overwrite this method in V2::BaseController that
+      # renders error if request.content_type != "application/json"
+      # the upload method uses "multipart/form-data"
```

Comment:
@daviddavis & I decided to use
```ruby
skip_before_filter :check_content_type, :only => [:upload]
```

That way we don't have to override the check_content_type method.